IntegrationTests/Services/AWSSTSIntegrationTests/STSWebIdentityAWSCredentialIdentityResolverTests.swift

```diff
@@ -61,15 +81,15 @@ class STSWebIdentityAWSCredentialIdentityResolverTests: XCTestCase {
     override func setUp() async throws {
         try await super.setUp()
 
+        // Create the Cognito identity pool that allows unauthenticated identities
+        try await createCognitoIdentityPool()
+
```

Comment:
The identity pool has to be created before the role is created, so that the identity pool ID can be specified in the assume role policy.  Hence, the order of steps is changed.